### ISSUE TITLE
Minimap: Terrain and owner layers lack contrast

### ISSUE THREAD

**Issue by Noordfrees** [widelands/widelands#4466]
The terrain layer can make it impossible to distinguish a field's owner on the minimap. Best seen with blue player and desert_water:
![shot0007](https://user-images.githubusercontent.com/48293446/98013541-c8091500-1dfa-11eb-8556-45243082b160.png)
The fields owned by the blue player are depicted in almost exactly the same colour regardless of whether the owner layer is switched on (left) or off (middle). The difference is not visible to the naked eye. The contrast needs to be increased.
